interfaces/cython/cantera/ck2yaml.py

```diff
@@ -865,6 +865,14 @@ def read_NASA7_entry(self, lines, TintDefault, comments):
         if not composition:
             raise InputError("Error parsing elemental composition for "
                              "species '{}'", species)
+        else:
+            for symbol in composition.keys():
+                # Here it checks if there is a number in the symbol string and if this is
+                # in the element list as a customized element
```

Comment:
The comment here explains *what* the line below it is doing, but not *why*. Most programmers could understand the *what* by just reading the code, so the comment isn't helping in that sense. Can you add a comment about *why* this change is needed instead?